pypom/page.py

```diff
@@ -18,11 +18,11 @@ class Page(WebView):
 
     Used as a base class for your project's page objects.
 
-    :param selenium: WebDriver object.
+    :param selenium: WebDriver like object.
     :param base_url: (optional) Base URL.
     :param timeout: (optional) Timeout used for explicit waits. Defaults to ``10``.
     :param url_kwargs: (optional) Keyword arguments used when formatting the :py:attr:`seed_url`.
-    :type selenium: :py:class:`~selenium.webdriver.remote.webdriver.WebDriver`
+    :type selenium: :py:class:`~selenium.webdriver.remote.webdriver.WebDriver` or any other supported driver
```

Comment:
Would it be useful to list the driver objects, so we can reference the Splinter docs too?